### DISCUSSION TITLE
Mishards docker image for Milvus(0.11.x) and pymilvus(0.3.x)

### DISCUSSION THREAD

**Discussion by stephenleo** [milvus-io/milvus#4711] [Q&A and General discussion] [answered]
Hi, thank you for the package. Current latest Mishards docker container only supports Milvus 0.10.5. 
Wanted to check if you have a plan to release an updated Mishards docker container image to use Milvus (0.11.x) and pymilvus (0.3.x)? 
Mainly to use the hybrid field capabilities from Milvus (0.11.x) and pymilvus (0.3.x).

**Comment by stephenleo** [comment, marked as answer]
ok found the answer that Milvus 0.11 does not support Mishards cluster solution yet: https://milvus.io/docs/v0.11.0/release_notes.md#RESTful

**Comment by yhmo** [comment]
Please read this announcement: https://github.com/milvus-io/milvus/discussions/4640
The 0.11.x is renamed to "experimental", and we will not maintain it. Currently the 0.10.x is the recommended version(we will continue maintain the 0.10.x). A new architect  milvus is in developing.
